README.md

```diff
@@ -1219,6 +1221,18 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Design Inspiration
```

Comment:
Very good, please add a blank line below the heading here and we're good to go! thanks for the great improvement and new entries